app/src/geckoBeta/java/org/mozilla/fenix/engine/GeckoProvider.kt

```diff
@@ -23,7 +23,7 @@ object GeckoProvider {
     @Synchronized
     fun getOrCreateRuntime(
         context: Context,
-        storage: LoginsStorage
+        storage: Lazy<LoginsStorage>
```

Comment:
You'll need to add these changes to the geckoNightly `GeckoProvider` too

Reply:
👍